specification/network/resource-manager/readme.md

```diff
@@ -95,6 +95,7 @@ input-file:
 - Microsoft.Network/2017-06-01/checkDnsAvailability.json
 - Microsoft.Network/2017-06-01/endpointService.json
 - Microsoft.Network/2017-06-01/expressRouteCircuit.json
+- Microsoft.Network/2017-06-01/expressRouteConnector.json
 - Microsoft.Network/2017-06-01/loadBalancer.json

```

Comment:
As discussed before, `expressRouteConnector.json` is missing in `2017-08-01` and `2017-09-01`. We need to confirm with Network team.